### DISCUSSION TITLE
Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by avi-chen-1** [PrincetonUniversity/athena#422] [General]
Hi, can you please help me with this. Been stuck for days.  
When i run the input file using HDF5 as output I get a segfault (which doesn't happen when output file is vtk). I downloaded and configured several versions of HDF5 following the simple instructions on this website: https://accserv.lepp.cornell.edu/svn/packages/hdf5/release_docs/INSTALL

I installed three different versions of HDF5 in three different folders but each time I run it, I get the segault above.

Any help would be greatly appreciated!
![Screenshot from 2022-01-14 17-57-32](https://user-images.githubusercontent.com/97767096/149595978-7e8a7ae2-2e9a-45ce-8dd8-1b9c9542d73e.png)

**Comment by tomidakn** [comment]
Please make sure you have the latest version of Athena++. A possibly-related bug was fixed on Dec 31st in commit 71c5bab .

**Reply by avi-chen-1** [reply]
Hey, thanks so much! This solved it in an instant :)
